server/routes/varyingLicences/handlers/caseload.test.ts

```diff
@@ -306,15 +306,80 @@ describe('Route Handlers - Vary Licence - Caseload', () => {
       caseloadService.getStaffVaryCaseload.mockResolvedValue([
         {
           licences: [
+            {
+              id: 1,
+              type: LicenceType.AP,
+              status: LicenceStatus.ACTIVE,
+            },
             {
```

Comment:
This test check to retrieve the non-active licence if a variation exist. 
Swapped the order here as it wasn't testing the code - it would have passed without special handling as it falls back to just return the first record otherwise